pulp-core/patches/content_parameter_filename_fix.patch

```diff
@@ -0,0 +1,19 @@
+diff --git a/pulpcore/content/handler.py b/pulpcore/content/handler.py
+index 26a8faa4c..6dad34d42 100644
+--- a/pulpcore/content/handler.py
++++ b/pulpcore/content/handler.py
+@@ -613,6 +613,14 @@ class Handler:
+             content_disposition = f"attachment;filename={content_artifact.relative_path}"
+             parameters = {"ResponseContentDisposition": content_disposition}
+             url = artifact_file.storage.url(artifact_file.name, parameters=parameters)
++            # workaround bug in django storages which ignores parameters
++            from urllib.parse import urlparse, urlunparse, parse_qs, urlencode
++            splitu = urlparse(url)
++            qs = parse_qs(splitu.query, keep_blank_values=True)
++            qs.update(parameters)
++            url = urlunparse(splitu._replace(query=urlencode(qs, doseq=True)))
```

Comment:
Should consider using `splitu._replace().geturl()` instead of passing in to urlunparse here? 🤷